### PR TITLE
Aml 1117 user api calls

### PR DESCRIPTION
**Pull request by robinoconnor**

### CODE REVIEW

File: src/SFA.DAS.EAS.Employer_Account.Database/SFA.DAS.EAS.Employer_Account.Database.sqlproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectVersion>4.1</ProjectVersion>
     <ProjectGuid>{927a0bea-b043-4a03-9311-f49cb60541d5}</ProjectGuid>
-    <DSP>Microsoft.Data.Tools.Schema.Sql.SqlAzureV12DatabaseSchemaProvider</DSP>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql120DatabaseSchemaProvider</DSP>
```
Comment (dashton82) [inline src/SFA.DAS.EAS.Employer_Account.Database/SFA.DAS.EAS.Employer_Account.Database.sqlproj]: Need to undo this change

**Comment by robinoconnor** [comment]
Changed
